### ISSUE TITLE
Private macros are reported as no longer exported

### ISSUE THREAD

**Issue by mtshiba** [obi1kenobi/cargo-semver-checks#1063]
### Which lint or lints are the issue

macro_no_longer_exported

### Known issues that might be causing this

- [ ] Is the flagged item defined in another crate, or a re-export of such an item? ([#638](https://github.com/obi1kenobi/cargo-semver-checks/issues/638))

### Steps to reproduce the bug with the above code

code: https://github.com/erg-lang/erg/blob/eedbfc27dc12f79d8a3a775830e3cca33da08d71/crates/erg_compiler/context/eval.rs#L43-L52
error log: https://github.com/erg-lang/erg/actions/runs/12532443587/job/34951178470

### Actual Behaviour

Macros are reported as “no longer exported” in the new version.

### Expected Behaviour

Macros are private in both the old and new versions, so this should not be an incompatible change.

### Verbose Lint Output

    Updating crates.io index
     Locking 83 packages to latest compatible versions
    Building erg_compiler v0.6.51-nightly.0 (current)
    Documenting erg_compiler v0.6.51-nightly.0
    Finished `dev` profile [unoptimized + debuginfo] target(s) in 5.54s
       Built [   5.633s] (current)
     Parsing erg_compiler v0.6.51-nightly.0 (current)
      Parsed [   0.037s] (current)
     Parsing erg_compiler v0.6.50 (baseline, cached)
      Parsed [   0.046s] (baseline)
    Checking erg_compiler v0.6.50 -> v0.6.51-nightly.0 (minor change)
    Starting 100 checks, 7 unnecessary on 8 threads
        PASS [   0.002s]       major        attribute_proc_macro_missing
        PASS [   0.038s]       major        auto_trait_impl_removed
        PASS [   0.010s]       major        constructible_struct_adds_field
        PASS [   0.004s]       major        constructible_struct_adds_private_field
        PASS [   0.001s]       major        constructible_struct_changed_type
        PASS [   0.012s]       major        declarative_macro_missing
        PASS [   0.002s]       major        derive_helper_attr_removed
        PASS [   0.001s]       major        derive_proc_macro_missing
        PASS [   0.071s]       major        derive_trait_impl_removed
        PASS [   0.006s]       major        enum_discriminants_undefined_non_exhaustive_variant
        PASS [   0.004s]       major        enum_discriminants_undefined_non_unit_variant
        PASS [   0.001s]       major        enum_marked_non_exhaustive
        PASS [   0.002s]       major        enum_missing
        PASS [   0.002s]       major        enum_no_repr_variant_discriminant_changed
        PASS [   0.003s]       major        enum_now_doc_hidden
        PASS [   0.002s]       major        enum_repr_int_changed
        PASS [   0.002s]       major        enum_repr_int_removed
        PASS [   0.001s]       major        enum_repr_transparent_removed
        PASS [   0.012s]       major        enum_repr_variant_discriminant_changed
        PASS [   0.013s]       major        enum_struct_variant_field_added
        PASS [   0.004s]       major        enum_struct_variant_field_missing
        PASS [   0.004s]       major        enum_struct_variant_field_now_doc_hidden
        PASS [   0.006s]       major        enum_tuple_variant_changed_kind
        PASS [   0.011s]       major        enum_tuple_variant_field_added
        PASS [   0.005s]       major        enum_tuple_variant_field_missing
        PASS [   0.009s]       major        enum_tuple_variant_field_now_doc_hidden
        PASS [   0.007s]       major        enum_unit_variant_changed_kind
        PASS [   0.015s]       major        enum_variant_added
        PASS [   0.022s]       major        enum_variant_marked_non_exhaustive
        PASS [   0.010s]       major        enum_variant_missing
        PASS [   0.001s]       major        exported_function_changed_abi
        PASS [   0.000s]       major        feature_missing
        PASS [   0.003s]       major        function_abi_no_longer_unwind
        PASS [   0.002s]       major        function_changed_abi
        PASS [   0.001s]       major        function_const_removed
        PASS [   0.001s]       major        function_export_name_changed
        PASS [   0.001s]       major        function_like_proc_macro_missing
        PASS [   0.005s]       major        function_missing
        PASS [   0.006s]       major        function_now_doc_hidden
        PASS [   0.006s]       major        function_parameter_count_changed
        PASS [   0.001s]       major        function_unsafe_added
        PASS [   0.002s]       major        inherent_associated_const_now_doc_hidden
        PASS [   0.002s]       major        inherent_associated_pub_const_missing
        PASS [   0.012s]       major        inherent_method_const_removed
        PASS [   0.070s]       major        inherent_method_missing
        PASS [   0.071s]       major        inherent_method_now_doc_hidden
        PASS [   0.073s]       major        inherent_method_unsafe_added
        FAIL [   0.017s]       major        macro_no_longer_exported
        PASS [   0.012s]       major        macro_now_doc_hidden
        PASS [   0.145s]       major        method_parameter_count_changed
        PASS [   0.005s]       major        module_missing
        PASS [   0.002s]       major        non_exhaustive_struct_changed_type
        PASS [   0.003s]       major        pub_module_level_const_missing
        PASS [   0.005s]       major        pub_module_level_const_now_doc_hidden
        PASS [   0.003s]       major        pub_static_missing
        PASS [   0.001s]       major        pub_static_mut_now_immutable
        PASS [   0.001s]       major        pub_static_now_doc_hidden
        PASS [   0.001s]       major        pub_static_now_mutable
        PASS [   0.001s]       major        repr_c_removed
        PASS [   0.002s]       major        repr_packed_added
        PASS [   0.001s]       major        repr_packed_removed
        PASS [   0.008s]       major        sized_impl_removed
        PASS [   0.003s]       major        struct_marked_non_exhaustive
        PASS [   0.010s]       major        struct_missing
        PASS [   0.003s]       major        struct_now_doc_hidden
        PASS [   0.006s]       major        struct_pub_field_missing
        PASS [   0.010s]       major        struct_pub_field_now_doc_hidden
        PASS [   0.004s]       major        struct_repr_transparent_removed
        PASS [   0.002s]       major        struct_with_no_pub_fields_changed_type
        PASS [   0.003s]       major        struct_with_pub_fields_changed_type
        PASS [   0.001s]       major        trait_added_supertrait
        PASS [   0.003s]       major        trait_associated_const_added
        PASS [   0.005s]       major        trait_associated_const_default_removed
        PASS [   0.004s]       major        trait_associated_const_now_doc_hidden
        PASS [   0.001s]       major        trait_associated_type_added
        PASS [   0.003s]       major        trait_associated_type_default_removed
        PASS [   0.001s]       major        trait_associated_type_now_doc_hidden
        PASS [   0.002s]       major        trait_method_added
        PASS [   0.001s]       major        trait_method_default_impl_removed
        PASS [   0.002s]       major        trait_method_missing
        PASS [   0.001s]       major        trait_method_now_doc_hidden
        PASS [   0.001s]       major        trait_method_unsafe_added
        PASS [   0.001s]       major        trait_method_unsafe_removed
        PASS [   0.001s]       major        trait_mismatched_generic_lifetimes
        PASS [   0.001s]       major        trait_missing
        PASS [   0.001s]       major        trait_newly_sealed
        PASS [   0.003s]       major        trait_no_longer_object_safe
        PASS [   0.004s]       major        trait_now_doc_hidden
        PASS [   0.004s]       major        trait_removed_associated_constant
        PASS [   0.001s]       major        trait_removed_associated_type
        PASS [   0.001s]       major        trait_removed_supertrait
        PASS [   0.001s]       major        trait_unsafe_added
        PASS [   0.001s]       major        trait_unsafe_removed
        PASS [   0.001s]       major        tuple_struct_to_plain_struct
        PASS [   0.006s]       major        type_mismatched_generic_lifetimes
        PASS [   0.001s]       major        union_field_missing
        PASS [   0.001s]       major        union_missing
        PASS [   0.001s]       major        union_now_doc_hidden
        PASS [   0.001s]       major        union_pub_field_now_doc_hidden
        PASS [   0.001s]       major        unit_struct_changed_kind
     Checked [   0.212s] 100 checks: 99 pass, 1 fail, 0 warn, 7 skip

--- failure macro_no_longer_exported: macro is no longer exported ---

Description:
A macro that was previously exported with #[macro_export] is no longer exported. This breaks downstream code that used the macro.
        ref: https://doc.rust-lang.org/reference/macros-by-example.html#path-based-scope
       impl: https://github.com/obi1kenobi/cargo-semver-checks/tree/v0.38.0/src/lints/macro_no_longer_exported.ron

Failed in:
  macro feature_error in erg/crates/erg_compiler/context/eval.rs:43
  macro unreachable_error in erg/crates/erg_compiler/context/eval.rs:48

     Summary semver requires new major version: 1 major and 0 minor checks failed
    Finished [   6.417s] erg_compiler

### Generated System Information

System information:
-------------------
#### Software version

cargo-semver-checks 0.38.0

#### Operating system

macOS 15.2 (Darwin 24.2.0)

#### Command-line

```bash
/Users/shiba/.cargo/bin/cargo-semver-checks semver-checks --bugreport 
```

#### cargo version

```
> cargo -V 
cargo 1.83.0 (5ffbef321 2024-10-29)
```

#### Compile time information

- Profile: release
- Target triple: aarch64-apple-darwin
- Family: unix
- OS: macos
- Architecture: aarch64
- Pointer width: 64
- Endian: little
- CPU features: aes,crc,dit,dotprod,dpb,dpb2,fcma,fhm,flagm,fp16,frintts,jsconv,lor,lse,neon,paca,pacg,pan,pmuv3,ras,rcpc,rcpc2,rdm,sb,sha2,sha3,ssbs,vh
- Host: aarch64-apple-darwin

Cargo build configuration:
--------------------------
[alias]
b = ["build"]
b_full = ["build", "--features", "full"]
b_full_ja = ["build", "--features", "full", "--features", "japanese"]
b_full_ja_re = ["build", "--features", "full", "--features", "japanese", "--release"]
b_full_re = ["build", "--features", "full", "--release"]
b_full_zh_cn = ["build", "--features", "full", "--features", "simplified_chinese"]
b_full_zh_cn_re = ["build", "--features", "full", "--features", "simplified_chinese", "--release"]
b_full_zh_tw = ["build", "--features", "full", "--features", "traditional_chinese"]
b_full_zh_tw_re = ["build", "--features", "full", "--features", "traditional_chinese", "--release"]
b_ja = ["build", "--features", "japanese"]
b_ja_re = ["build", "--features", "japanese", "--release"]
b_re = ["build", "--release"]
b_zh_cn = ["build", "--features", "simplified_chinese"]
b_zh_cn_re = ["build", "--features", "simplified_chinese", "--release"]
b_zh_tw = ["build", "--features", "traditional_chinese"]
b_zh_tw_re = ["build", "--features", "traditional_chinese", "--release"]
bd = ["build", "--features", "debug"]
bd_ja = ["build", "--features", "debug", "--features", "japanese"]
bd_ja_re = ["build", "--features", "debug", "--features", "japanese", "--release"]
bd_re = ["build", "--features", "debug", "--release"]
bd_zh_cn = ["build", "--features", "debug", "--features", "simplified_chinese"]
bd_zh_cn_re = ["build", "--features", "debug", "--features", "simplified_chinese", "--release"]
bd_zh_tw = ["build", "--features", "debug", "--features", "traditional_chinese"]
bd_zh_tw_re = ["build", "--features", "debug", "--features", "traditional_chinese", "--release"]
chk = ["run", "--", "--mode", "check"]
cmp = ["run", "--", "--mode", "compile"]
dchk = ["run", "--features", "debug", "--", "--mode", "check"]
dcmp = ["run", "--features", "debug", "--", "--mode", "compile"]
dex = ["run", "--features", "debug", "--", "--mode", "exec"]
dinst = ["install", "--path", ".", "--features", "debug", "--features", "els"]
dlex = ["run", "--features", "debug", "--", "--mode", "lex"]
dlwr = ["run", "--features", "debug", "--", "--mode", "lower"]
dprs = ["run", "--features", "debug", "--", "--mode", "parse"]
drc_r = ["r", "-Zbuild-std", "-Zbuild-std-features=core/debug_refcell"]
dread = ["run", "--features", "debug", "--", "--mode", "read"]
dtrans = ["run", "--features", "debug", "--", "--mode", "transpile"]
ex = ["run", "--", "--mode", "exec"]
inst = ["install", "--path", ".", "--features", "els"]
lex = ["run", "--", "--mode", "lex"]
lwr = ["run", "--", "--mode", "lower"]
ntest = ["nextest", "run"]
prs = ["run", "--", "--mode", "parse"]
r = ["run"]
r_full = ["run", "--features", "full"]
r_full_ja = ["run", "--features", "full", "--features", "japanese"]
r_full_ja_re = ["run", "--features", "full", "--features", "japanese", "--release"]
r_full_re = ["run", "--features", "full", "--release"]
r_full_zh_cn = ["run", "--features", "full", "--features", "simplified_chinese"]
r_full_zh_cn_re = ["run", "--features", "full", "--features", "simplified_chinese", "--release"]
r_full_zh_tw = ["run", "--features", "full", "--features", "traditional_chinese"]
r_full_zh_tw_re = ["run", "--features", "full", "--features", "traditional_chinese", "--release"]
r_ja = ["run", "--features", "japanese"]
r_ja_re = ["run", "--features", "japanese", "--release"]
r_re = ["run", "--release"]
r_zh_cn = ["run", "--features", "simplified_chinese"]
r_zh_cn_re = ["run", "--features", "simplified_chinese", "--release"]
r_zh_tw = ["run", "--features", "traditional_chinese"]
r_zh_tw_re = ["run", "--features", "traditional_chinese", "--release"]
rd = ["run", "--features", "debug"]
rd_ja = ["run", "--features", "debug", "--features", "japanese"]
rd_ja_re = ["run", "--features", "debug", "--features", "japanese", "--release"]
rd_re = ["run", "--features", "debug", "--release"]
rd_zh_cn = ["run", "--features", "debug", "--features", "simplified_chinese"]
rd_zh_cn_re = ["run", "--features", "debug", "--features", "simplified_chinese", "--release"]
rd_zh_tw = ["run", "--features", "debug", "--features", "traditional_chinese"]
rd_zh_tw_re = ["run", "--features", "debug", "--features", "traditional_chinese", "--release"]
read = ["run", "--", "--mode", "read"]
trans = ["run", "--", "--mode", "transpile"]

Please file an issue on GitHub reporting your bug.
Consider adding the diagnostic information above, either manually or automatically through the link below:

https://github.com/obi1kenobi/cargo-semver-checks/issues/new?labels=C-bug&template=3-bug-report.yml&sys-info=%23%23%23%23%20Software%20version%0A%0Acargo-semver-checks%200.38.0%0A%0A%23%23%23%23%20Operating%20system%0A%0AmacOS%2015.2%20%28Darwin%2024.2.0%29%0A%0A%23%23%23%23%20Command-line%0A%0A%60%60%60bash%0A%2FUsers%2Fshiba%2F.cargo%2Fbin%2Fcargo-semver-checks%20semver-checks%20--bugreport%20%0A%60%60%60%0A%0A%23%23%23%23%20cargo%20version%0A%0A%60%60%60%0A%3E%20cargo%20-V%20%0Acargo%201.83.0%20%285ffbef321%202024-10-29%29%0A%60%60%60%0A%0A%23%23%23%23%20Compile%20time%20information%0A%0A-%20Profile%3A%20release%0A-%20Target%20triple%3A%20aarch64-apple-darwin%0A-%20Family%3A%20unix%0A-%20OS%3A%20macos%0A-%20Architecture%3A%20aarch64%0A-%20Pointer%20width%3A%2064%0A-%20Endian%3A%20little%0A-%20CPU%20features%3A%20aes%2Ccrc%2Cdit%2Cdotprod%2Cdpb%2Cdpb2%2Cfcma%2Cfhm%2Cflagm%2Cfp16%2Cfrintts%2Cjsconv%2Clor%2Clse%2Cneon%2Cpaca%2Cpacg%2Cpan%2Cpmuv3%2Cras%2Crcpc%2Crcpc2%2Crdm%2Csb%2Csha2%2Csha3%2Cssbs%2Cvh%0A-%20Host%3A%20aarch64-apple-darwin%0A%0A&build-config=%5Balias%5D%0Ab%20%3D%20%5B%22build%22%5D%0Ab_full%20%3D%20%5B%22build%22%2C%20%22--features%22%2C%20%22full%22%5D%0Ab_full_ja%20%3D%20%5B%22build%22%2C%20%22--features%22%2C%20%22full%22%2C%20%22--features%22%2C%20%22japanese%22%5D%0Ab_full_ja_re%20%3D%20%5B%22build%22%2C%20%22--features%22%2C%20%22full%22%2C%20%22--features%22%2C%20%22japanese%22%2C%20%22--release%22%5D%0Ab_full_re%20%3D%20%5B%22build%22%2C%20%22--features%22%2C%20%22full%22%2C%20%22--release%22%5D%0Ab_full_zh_cn%20%3D%20%5B%22build%22%2C%20%22--features%22%2C%20%22full%22%2C%20%22--features%22%2C%20%22simplified_chinese%22%5D%0Ab_full_zh_cn_re%20%3D%20%5B%22build%22%2C%20%22--features%22%2C%20%22full%22%2C%20%22--features%22%2C%20%22simplified_chinese%22%2C%20%22--release%22%5D%0Ab_full_zh_tw%20%3D%20%5B%22build%22%2C%20%22--features%22%2C%20%22full%22%2C%20%22--features%22%2C%20%22traditional_chinese%22%5D%0Ab_full_zh_tw_re%20%3D%20%5B%22build%22%2C%20%22--features%22%2C%20%22full%22%2C%20%22--features%22%2C%20%22traditional_chinese%22%2C%20%22--release%22%5D%0Ab_ja%20%3D%20%5B%22build%22%2C%20%22--features%22%2C%20%22japanese%22%5D%0Ab_ja_re%20%3D%20%5B%22build%22%2C%20%22--features%22%2C%20%22japanese%22%2C%20%22--release%22%5D%0Ab_re%20%3D%20%5B%22build%22%2C%20%22--release%22%5D%0Ab_zh_cn%20%3D%20%5B%22build%22%2C%20%22--features%22%2C%20%22simplified_chinese%22%5D%0Ab_zh_cn_re%20%3D%20%5B%22build%22%2C%20%22--features%22%2C%20%22simplified_chinese%22%2C%20%22--release%22%5D%0Ab_zh_tw%20%3D%20%5B%22build%22%2C%20%22--features%22%2C%20%22traditional_chinese%22%5D%0Ab_zh_tw_re%20%3D%20%5B%22build%22%2C%20%22--features%22%2C%20%22traditional_chinese%22%2C%20%22--release%22%5D%0Abd%20%3D%20%5B%22build%22%2C%20%22--features%22%2C%20%22debug%22%5D%0Abd_ja%20%3D%20%5B%22build%22%2C%20%22--features%22%2C%20%22debug%22%2C%20%22--features%22%2C%20%22japanese%22%5D%0Abd_ja_re%20%3D%20%5B%22build%22%2C%20%22--features%22%2C%20%22debug%22%2C%20%22--features%22%2C%20%22japanese%22%2C%20%22--release%22%5D%0Abd_re%20%3D%20%5B%22build%22%2C%20%22--features%22%2C%20%22debug%22%2C%20%22--release%22%5D%0Abd_zh_cn%20%3D%20%5B%22build%22%2C%20%22--features%22%2C%20%22debug%22%2C%20%22--features%22%2C%20%22simplified_chinese%22%5D%0Abd_zh_cn_re%20%3D%20%5B%22build%22%2C%20%22--features%22%2C%20%22debug%22%2C%20%22--features%22%2C%20%22simplified_chinese%22%2C%20%22--release%22%5D%0Abd_zh_tw%20%3D%20%5B%22build%22%2C%20%22--features%22%2C%20%22debug%22%2C%20%22--features%22%2C%20%22traditional_chinese%22%5D%0Abd_zh_tw_re%20%3D%20%5B%22build%22%2C%20%22--features%22%2C%20%22debug%22%2C%20%22--features%22%2C%20%22traditional_chinese%22%2C%20%22--release%22%5D%0Achk%20%3D%20%5B%22run%22%2C%20%22--%22%2C%20%22--mode%22%2C%20%22check%22%5D%0Acmp%20%3D%20%5B%22run%22%2C%20%22--%22%2C%20%22--mode%22%2C%20%22compile%22%5D%0Adchk%20%3D%20%5B%22run%22%2C%20%22--features%22%2C%20%22debug%22%2C%20%22--%22%2C%20%22--mode%22%2C%20%22check%22%5D%0Adcmp%20%3D%20%5B%22run%22%2C%20%22--features%22%2C%20%22debug%22%2C%20%22--%22%2C%20%22--mode%22%2C%20%22compile%22%5D%0Adex%20%3D%20%5B%22run%22%2C%20%22--features%22%2C%20%22debug%22%2C%20%22--%22%2C%20%22--mode%22%2C%20%22exec%22%5D%0Adinst%20%3D%20%5B%22install%22%2C%20%22--path%22%2C%20%22.%22%2C%20%22--features%22%2C%20%22debug%22%2C%20%22--features%22%2C%20%22els%22%5D%0Adlex%20%3D%20%5B%22run%22%2C%20%22--features%22%2C%20%22debug%22%2C%20%22--%22%2C%20%22--mode%22%2C%20%22lex%22%5D%0Adlwr%20%3D%20%5B%22run%22%2C%20%22--features%22%2C%20%22debug%22%2C%20%22--%22%2C%20%22--mode%22%2C%20%22lower%22%5D%0Adprs%20%3D%20%5B%22run%22%2C%20%22--features%22%2C%20%22debug%22%2C%20%22--%22%2C%20%22--mode%22%2C%20%22parse%22%5D%0Adrc_r%20%3D%20%5B%22r%22%2C%20%22-Zbuild-std%22%2C%20%22-Zbuild-std-features%3Dcore%2Fdebug_refcell%22%5D%0Adread%20%3D%20%5B%22run%22%2C%20%22--features%22%2C%20%22debug%22%2C%20%22--%22%2C%20%22--mode%22%2C%20%22read%22%5D%0Adtrans%20%3D%20%5B%22run%22%2C%20%22--features%22%2C%20%22debug%22%2C%20%22--%22%2C%20%22--mode%22%2C%20%22transpile%22%5D%0Aex%20%3D%20%5B%22run%22%2C%20%22--%22%2C%20%22--mode%22%2C%20%22exec%22%5D%0Ainst%20%3D%20%5B%22install%22%2C%20%22--path%22%2C%20%22.%22%2C%20%22--features%22%2C%20%22els%22%5D%0Alex%20%3D%20%5B%22run%22%2C%20%22--%22%2C%20%22--mode%22%2C%20%22lex%22%5D%0Alwr%20%3D%20%5B%22run%22%2C%20%22--%22%2C%20%22--mode%22%2C%20%22lower%22%5D%0Antest%20%3D%20%5B%22nextest%22%2C%20%22run%22%5D%0Aprs%20%3D%20%5B%22run%22%2C%20%22--%22%2C%20%22--mode%22%2C%20%22parse%22%5D%0Ar%20%3D%20%5B%22run%22%5D%0Ar_full%20%3D%20%5B%22run%22%2C%20%22--features%22%2C%20%22full%22%5D%0Ar_full_ja%20%3D%20%5B%22run%22%2C%20%22--features%22%2C%20%22full%22%2C%20%22--features%22%2C%20%22japanese%22%5D%0Ar_full_ja_re%20%3D%20%5B%22run%22%2C%20%22--features%22%2C%20%22full%22%2C%20%22--features%22%2C%20%22japanese%22%2C%20%22--release%22%5D%0Ar_full_re%20%3D%20%5B%22run%22%2C%20%22--features%22%2C%20%22full%22%2C%20%22--release%22%5D%0Ar_full_zh_cn%20%3D%20%5B%22run%22%2C%20%22--features%22%2C%20%22full%22%2C%20%22--features%22%2C%20%22simplified_chinese%22%5D%0Ar_full_zh_cn_re%20%3D%20%5B%22run%22%2C%20%22--features%22%2C%20%22full%22%2C%20%22--features%22%2C%20%22simplified_chinese%22%2C%20%22--release%22%5D%0Ar_full_zh_tw%20%3D%20%5B%22run%22%2C%20%22--features%22%2C%20%22full%22%2C%20%22--features%22%2C%20%22traditional_chinese%22%5D%0Ar_full_zh_tw_re%20%3D%20%5B%22run%22%2C%20%22--features%22%2C%20%22full%22%2C%20%22--features%22%2C%20%22traditional_chinese%22%2C%20%22--release%22%5D%0Ar_ja%20%3D%20%5B%22run%22%2C%20%22--features%22%2C%20%22japanese%22%5D%0Ar_ja_re%20%3D%20%5B%22run%22%2C%20%22--features%22%2C%20%22japanese%22%2C%20%22--release%22%5D%0Ar_re%20%3D%20%5B%22run%22%2C%20%22--release%22%5D%0Ar_zh_cn%20%3D%20%5B%22run%22%2C%20%22--features%22%2C%20%22simplified_chinese%22%5D%0Ar_zh_cn_re%20%3D%20%5B%22run%22%2C%20%22--features%22%2C%20%22simplified_chinese%22%2C%20%22--release%22%5D%0Ar_zh_tw%20%3D%20%5B%22run%22%2C%20%22--features%22%2C%20%22traditional_chinese%22%5D%0Ar_zh_tw_re%20%3D%20%5B%22run%22%2C%20%22--features%22%2C%20%22traditional_chinese%22%2C%20%22--release%22%5D%0Ard%20%3D%20%5B%22run%22%2C%20%22--features%22%2C%20%22debug%22%5D%0Ard_ja%20%3D%20%5B%22run%22%2C%20%22--features%22%2C%20%22debug%22%2C%20%22--features%22%2C%20%22japanese%22%5D%0Ard_ja_re%20%3D%20%5B%22run%22%2C%20%22--features%22%2C%20%22debug%22%2C%20%22--features%22%2C%20%22japanese%22%2C%20%22--release%22%5D%0Ard_re%20%3D%20%5B%22run%22%2C%20%22--features%22%2C%20%22debug%22%2C%20%22--release%22%5D%0Ard_zh_cn%20%3D%20%5B%22run%22%2C%20%22--features%22%2C%20%22debug%22%2C%20%22--features%22%2C%20%22simplified_chinese%22%5D%0Ard_zh_cn_re%20%3D%20%5B%22run%22%2C%20%22--features%22%2C%20%22debug%22%2C%20%22--features%22%2C%20%22simplified_chinese%22%2C%20%22--release%22%5D%0Ard_zh_tw%20%3D%20%5B%22run%22%2C%20%22--features%22%2C%20%22debug%22%2C%20%22--features%22%2C%20%22traditional_chinese%22%5D%0Ard_zh_tw_re%20%3D%20%5B%22run%22%2C%20%22--features%22%2C%20%22debug%22%2C%20%22--features%22%2C%20%22traditional_chinese%22%2C%20%22--release%22%5D%0Aread%20%3D%20%5B%22run%22%2C%20%22--%22%2C%20%22--mode%22%2C%20%22read%22%5D%0Atrans%20%3D%20%5B%22run%22%2C%20%22--%22%2C%20%22--mode%22%2C%20%22transpile%22%5D%0A

**Comment by obi1kenobi** [comment]
Oof, I'm sorry about this bug. I'm looking into it right now.

In the meantime, if you'd like you can suppress this lint by adding the following to your workspace configuration:
```toml
[workspace.metadata.cargo-semver-checks.lints]
macro_no_longer_exported = "allow"
```

**Comment by obi1kenobi** [comment]
Ah it appears I've already squashed this bug in my recent macro exports rework. The fix is merged and just not published yet.

I'm currently on a family vacation, but I'll make a new release as soon as I'm back home in about a week or so.

In the meantime, I recommend either using the config workaround above to disable the buggy lint, or using `cargo install --git https://github.com/obi1kenobi/cargo-semver-checks.git` to install directly from the repo.
